nms/app/packages/magmalte/app/views/subscriber/__tests__/SubscriberAddEditTest.js

```diff
@@ -277,72 +284,72 @@ describe('<AddSubscriberButton />', () => {
     );
   };
 
-  const DetailWrapper = () => {
-    const [subscribers, setSubscribers] = useState(subscribersMock);
-    const policyCtx = {
-      state: policies,
-      qosProfiles: {},
-      ratingGroups: {},
-      setRatingGroups: async () => {},
-      setQosProfiles: async () => {},
-      setState: async () => {},
-    };
+  // const DetailWrapper = () => {
```

Comment:
Can you add an issue to fix this test and add that issue as a comment here and assign it to yourself

Reply:
https://github.com/magma/magma/issues/4405